include/verilated.cpp

```diff
@@ -2520,12 +2520,25 @@ void VerilatedContextImp::commandArgVl(const std::string& arg) {
                         "Exiting due to command line argument (not an error)");
         } else if (arg == "+verilator+noassert") {
             assertOn(false);
-        } else if (commandArgVlValue(arg, "+verilator+prof+threads+start+", value /*ref*/)) {
-            profThreadsStart(std::atoll(value.c_str()));
-        } else if (commandArgVlValue(arg, "+verilator+prof+threads+window+", value /*ref*/)) {
-            profThreadsWindow(std::atol(value.c_str()));
-        } else if (commandArgVlValue(arg, "+verilator+prof+threads+file+", value /*ref*/)) {
-            profThreadsFilename(value);
+        } else if (commandArgVlValue(arg, "+verilator+prof+threads+start+", value /*ref*/)
+                   || commandArgVlValue(arg, "+verilator+prof+exec+start+", value /*ref*/)) {
+            const long long int val = std::atoll(value.c_str());
+            if (val < 0) {
+                VL_FATAL_MT("COMMAND_LINE", 0, "",
+                            "+verilator+prof+exec+start argument must be a non-negative integer");
+            }
```

Comment:
Maybe better to make a commandArgVlValuePositive?   Or perhaps something that can be used both here and below that has arg if zero is allowed?

If adding these tests also add a test case please so code coverage doesn't go down.

Reply:
Done, will push this separately.